humhub/protected/config/common.php

```diff
@@ -14,5 +15,45 @@
             'showScriptName' => false,
             'enablePrettyUrl' => true,
         ],
+        
     ]
 ];
+
+/**
+ * Redis configuration.
+ * 
+ * @see https://docs.humhub.org/docs/admin/redis
+ */
+if (!empty(getenv('HUMHUB_REDIS_HOSTNAME'))) {
```

Comment:
I have no good feeling having this much logic and calls in the config. It seems rather unusual and could add performance problems. Did you look at the humhub config system? Is this array cached somehow or will there be getenv calls on every request?

Reply:
From what I've seen, this is executed on each PHP requests ie. not cached.

This is clearly not the best solution, which is why it's still in draft, but it allowed me to test if HumHub could read config from env var. Unfortunately, env var set at the container level are not visible from the process started by supervisor, thus the in progress modification to replace those values.